content/concepts/the-rippled-server/transaction-censorship-detection.md

```diff
@@ -0,0 +1,42 @@
+# Transaction Censorship Detection
+
+The XRP Ledger is designed to be [censorship resistant](xrp-ledger-overview.html#censorship-resistant-transaction-processing). Any attempt to censor transactions requires coordinated action by a majority of the XRP Ledger network's validators.
+
+XRP Ledger provides an automated transaction censorship detector that is available on all `rippled` servers, enabling all participants to see if censorship is affecting the network.
+
+While a `rippled` server is in sync with the network, the detector tracks all transactions that, in the view of the `rippled` server, should have been accepted in the last round of [consensus](intro-to-consensus.html) and included in the last validated ledger. The detector issues log messages of increasing severity for transactions that have not been included in a validated ledger after several rounds of consensus.
+
+A `rippled` server is considered to be in sync if it meets the following criteria: ***TODO: Is this info useful or overkill?***
```

Comment:
I think maybe overkill, especially since some of these points are prerequisites for others.